firestore-translate-text/extension.yaml

```diff
@@ -132,26 +132,64 @@ params:
     required: false
 
   - param: TRANSLATION_MODEL
-    label: Translation Model
+    label: Translation Engine
```

Comment:
Wouldn't `Translation Model` work better given the param is `TRANSLATION_MODEL`. Also considering the file's param-label naming convention?